mandatory/1-syntax-errors.js

```diff
@@ -19,24 +19,22 @@ function getTotal(a, b) {
 
 There are some Tests in this file that will help you work out if your code is working.
 
-To run these tests type `node 1-syntax-errors.js` into your terminal
+To run the tests for just this one file, type `npm test -- --testPathPattern 1-syntax-errors` into your terminal
```

Comment:
Isn't `--testPathPattern` effectively the default for positional args? `npm test -- 1-syntax-errors` would work too.

Reply:
I thought so too, but when I tried it it still ran all of the tests:

```console
% npm test -- 1-currency-conversion

> javascript-core-1-coursework-week1@1.0.0 test
> jest --testRegex='mandatory[/\\].*\.js$' "1-currency-conversion"

 FAIL  mandatory/2-logic-error.js
[...snipped...]
```